docs/modules/api/pages/options.adoc

```diff
@@ -53,13 +55,13 @@ _(Experimental)._
 |`erb`, `erubis`
 
 |`:extensions`
-|A Ruby block that registers (and possibly defines) xref:extensions:register.adoc[Asciidoctor extensions] for this instance of the processor.
+|Specifies a xref:extensions:register.adoc#groups[processor extension group] to be registered globally for this instance of the processor.
 |_not set_
-|A Ruby block that conforms to the Asciidoctor extensions API (the same code that would be passed to the `Extensions.register` method).
+|A Ruby block, class, or instance that provides a processor extension group (the same code that would be passed to the `Extensions.register` method).
 
 |`:extension_registry`
-|Overrides the extensions registry instance.
-Instead of providing a Ruby block containing extensions to register, this option lets you replace the extension registry itself, giving you complete control over how extensions are registered for this processor.
+|Specifies an explicit xref:extensions:glossary.adoc#id_registry[extensions registry] instance.
+Instead of providing a processor extension group to register extensions, using the :extensions option, this option lets you replace the extension registry itself, giving you complete control over how extensions are registered for this processor.
```

Comment:
> using the :extensions option, this option lets...

We should probably try to avoid this repetition

Reply:
Rewritten extensively.